mafiasi/jabber/models.py

```diff
@@ -1,4 +1,4 @@
-from cgi import escape
```

Comment:
`cgi.escape` is deprecated and removed in python 3.8 (https://docs.python.org/3.7/library/cgi.html#cgi.escape)